lib/internal/module.js

```diff
@@ -130,3 +121,12 @@ function addBuiltinLibsToObject(object) {
     });
   });
 }
+
+module.exports = {
```

Comment:
that particular bit could likely be refactored but I'll just replace the `= exports =` here for now